resources/views/company/index.blade.php

```diff
@@ -108,7 +108,7 @@
               </div>
             </div>
            <table id="myTable" class="table table-hover table-nowrap table-bordered shadow-sm"  width="100%" ></table>
-            <select id="filter-country" class="form-select form-control form-control-sm py-1 px-1 d-none" style="width: 200px" aria-label="Default select example">
+            <select id="filter-country" class="form-select form-control form-control-sm py-0 px-1 d-none" style="width: 150px" aria-label="Default select example">
```

Comment:
use bootstrap class instead of hardcoding inline css

Reply:
Resolved